### ISSUE TITLE
View multiple streams in tiled windows

### ISSUE THREAD

**Issue by MWers** [kilianc/rtail#106]
Is it possible to view multiple streams at once in a tiled display? I'd love to use rtail to recreate something like the following:

<img width="1135" alt="2__matt_matthews-macbook-pro_____zsh_" src="https://cloud.githubusercontent.com/assets/808995/12104462/73643fb2-b300-11e5-8e86-e80a3cea713c.png">

Is that what this issue references? 

https://github.com/kilianc/rtail/issues/29

**Comment by kilianc** [comment]
@MWers yes it is, we have little to no data on this particular use case and we're trying to understand the best approach, I'll keep this open and let you know as soon as there's something to discuss.

thanks for stopping by!
